docs/versioned_docs/version-v0.86.0/developers/guides/local_env/getting_started_on_testnet.md

```diff
@@ -37,12 +37,12 @@ bash -i <(curl -s https://install.aztec.network)
 Then install the version of the network running the testnet:
 
 ```bash
-aztec-up 0.85.0-alpha-testnet.3
+aztec-up 0.85.0-alpha-testnet.5
```

Comment:
We really should make the testnet version a snippet `0.85.0-alpha-testnet.n`, and include that wherever needed.
Then we only need to edit `n` in one place, and it would be shared between versions.

Reply:
added a snippet. it isn't rendering properly in some code blocks, so i removed the places where i used it so we can get this merged